modules/schema/src/category/common.ts

```diff
@@ -14,17 +14,31 @@ export type Batch = {
   recordBatch?: RecordBatch;
   length: number;
   schema?: Schema;
-  bytesUsed?: number;
-  count?: number;
-  cursor?: number;
-  [key: string]: any;
-};
 
-/*
-export type Batch = {
+  /** Bytes processed in all batches up to and including this batch */
   bytesUsed?: number;
+
+  /**
+   * Total number of bytes in all batches
+   *
+   * This field is not always available, for instance when fetching compressed data
+   */
+  bytesTotal?: number;
+
+  /**
+   * Total number of rows parsed up to and including this batch
+   */
+  rowsUsed?: number;
+
+  /** Total number of rows in all batches
+   *
+   * This field is not always available
+   */
+  rowsTotal?: number;
+
+  /** @deprecated Use bytesUsed or rowsUsed */
   count?: number;
+
+  /** @deprecated */
```

Comment:
Use bytesUsed?

Reply:
I wasn't sure what `cursor` was alluding to